02/app.js

```diff
@@ -1,7 +1,24 @@
+// /* rozwiązanie z pętlą for */
+// const x = prompt("Podaj liczbę!");
 
-/* rozwiązanie z pętlą for */
-const x = 4;
+// if (x > 9) {
+// 	console.log("Za duża liczba");
+// } else {
+// 	for (let i = 1; i <= 9; i++) {
+// 		let result = x * i;
+// 		console.log(`${x} * ${i} = ${result}`);
+// 	}
+// }
 
+/* rozwiązanie z pętlą while  */
 
+let a = 3;
+let n = 4;
 
-/* rozwiązanie z pętlą while  */
\ No newline at end of file
+while (a <= 9 && a >= 1 && n <= 9 && n >= 1) {
+	let result = a ** n;
```

Comment:
Tutaj musisz użyć dodatkowej zmiennej, która w pętli będzie uzupełniana czyli pętla `for`, która będzie wykonywana n razy. Dzięki temu do tej zmiennej będzie przypisywany element `a` odpowiednią ilość razy coś w stylu:
```
let str = ''
for(...) {
    // tutaj będzie jeszcze potrzebny jakis if - ale to już na końcu do wymyślenia
    str = str + a;
    
}
console.log(str);
```